make/toolchain-android.cmake

```diff
@@ -47,9 +47,7 @@ set(CMAKE_SYSTEM_VERSION 1)
 if (NOT DEFINED TARGET_ABI)
   set(TARGET_ABI "arm-linux-androideabi")
 endif ()
-if (TARGET_ABI MATCHES "^arm")
-  set(CMAKE_SYSTEM_PROCESSOR arm)
-endif ()
+set(CMAKE_SYSTEM_PROCESSOR arm)
```

Comment:
Ok I think the problem is that SYS_rt_sigqueueinfo is not defined in syscall.h shipped with the Ubuntu package gcc-arm-linux-androideabi